operations-stdlib/src/commonTest/kotlin/string/ReplaceTest.kt

```diff
@@ -0,0 +1,96 @@
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+import JsonLogicResult.Success
+import JsonLogicResult.Failure
+import string.Replace
+
+class ReplaceTest:FunSpec({
```

Comment:
adding some more test cases where you use data and/or complex expressions with some other operations(var for example) would be great